git-work.sh

```diff
@@ -31,47 +31,46 @@ usage() {
   echo "Available subcommands are:"
   echo "   issue <id>              Start a new branch from issue <id>"
   echo "   commit <\"msg\">          Commit current work using <message>"
-  echo "   done <id>               Finish current work and merge"
-  echo "   push <master>           Push to the server"
+  echo "   done                    Finish current work and merge"
+  echo "   push                    Push to the server"
   echo "   release <tag> [<\"msg\">] Release a new tag called <tag> with <message>"
   echo "   pr <id>                 Create a branch from a pull-request"
 }
 
-if [ -z $1 ] || [ -z $2 ]; then
-  usage
-  exit 1
-fi
-
 case "$1" in
   # start a new branch
   issue)
+    [ -z $2 ] && ( usage && exit 1 )
     git checkout -b "$2"
     ;;
   # commit current work
   commit)
+    [ -z $2 ] && ( usage && exit 1 )
     git commit -am "$2"
     ;;
   # finish current work and merge
   done)
-    git checkout master
-    git merge "$2"
+    branch=${2:-$(git symbolic-ref --short HEAD)}
+    git merge "$branch" master
     ;;
   # push to the server
   push)
-    git push origin "$2"
+    branch=${2:-$(git symbolic-ref --short HEAD)}
+    git push origin "$branch"
     ;;
   # release a new tag
   release)
-    git checkout master
+    [ -z $2 ] && ( usage && exit 1 )
```

Comment:
I didn't test, but I believe that will be the same error https://github.com/mdbs99/git-work/pull/4/commits/5a27305364349f3f584563cf9343a3319e0c1443?diff=split#r131385035

Try to run both commands on `master`.